scripts/release/release-mrc.sh

```diff
@@ -0,0 +1,135 @@
+    exit 1
+  fi
+
+  if [[ "${GIT_BRANCH}" != "master" && ! "${DRY_RELEASE}" ]]; then
+    printf "%s\n" "Only dry releases are allowed on side branches"
+    exit 1
+  fi
+
+  tag="$1"
+
+  if "${DRY_RELEASE}"; then
+    printf "%s\n" "Dry releasing"
+    node_modules/.bin/lerna version --scope=@ovh-ux/manager-react-components --conventional-commits --no-commit-hooks --no-git-tag-version --no-push --allow-branch="${GIT_BRANCH}" --yes
+  else
+    printf "%s\n" "Releasing"
+    node_modules/.bin/lerna version --conventional-commits --no-commit-hooks --no-git-tag-version --no-push --no-private --ignore-changes "**/packages/manager/modules/manager-pci-common/**" --ignore-changes @ovh-ux/manager-pci-common --yes
+  fi
+}
+
+get_changed_packages() {
+  node_modules/.bin/lerna changed --all -p -l
+}
+
+get_release_name() {
+  seed="$1"
+  release_name=$(node scripts/release/index.js "$seed")
+  latest_release="$(git tag -l "${release_name}*" --sort=taggerdate | tail -1)"
+  if [ -z "$latest_release" ]; then
+    printf "%s\n" "$release_name"
+  else
+    id="${latest_release/$release_name-/}"
+    if [ -z "$id" ]; then
+      printf "%s\n" "$release_name-1"
+    else
+      next_id="$((id + 1))"
+      printf "%s\n" "$release_name-$next_id"
+    fi
+  fi
+}
+
+create_release_note() (
+  node_modules/.bin/conventional-changelog -p angular -n scripts/release/changelog.js -k "$1" --commit-path "$1" -l "$2"
+)
+
+push_and_release() {
+  printf "%s\n" "Commit and tag"
+  git git add packages/manager-react-components/package.json packages/manager-react-components/CHANGELOG.md
+  git commit -s -m "release: $1"
+  git tag -a -m "release: $1" "$1"
+}
+
+update_sonar_version() {
+  printf "%s\n" "Updating sonar"
+  sed -i "s/sonar\.projectVersion=.*/sonar\.projectVersion=$1/" ".sonarcloud.properties"
+}
+
+help()
+{
+   # Display Help
+   echo "Release monorepository"
+   echo
+   echo "options:"
+   echo "d     Dry release. Create a release in local without pushing or tagging to remote"
+   echo "s     Seed. Specify seed to generate release name"
+   echo "h     help."
+   echo
+}
+
+main() {
+  # Parse options
+  while getopts "dsh" option; do
+    case "${option}" in
+      d)
+        DRY_RELEASE=true
+        ;;
+      s)
+        SEED="${OPTARG}"
+        ;;
+      h)
+        help
+        ;;
+      *)
+        help
+        ;;
+    esac
+  done
+
+  changed_packages=$(get_changed_packages)
+  if [ -z "$changed_packages" ]; then
+    printf "%s\n" "Nothing to release"
+    exit 0
+  fi
+
+  current_tag="$(git describe --abbrev=0)"
+  printf "%s\n" "Previous tag was $current_tag"
+
+  # Separate versioning for `manager-react-components` and other packages
+  while read -r package; do
+    # Check if the changed package is `manager-react-components`
+    if [[ "$package" == *manager-react-components* ]]; then
+      printf "%s\n" "New release for manager-react-components"
+      next_tag=$(get_release_name "$SEED")
```

Comment:
`get_release_name` cannot be used here as it directly collides with actual manager releases pushed at https://github.com/ovh/manager/releases. So release name has to be `@ovh-ux/manager-react-components@1.x.x` where 1.x.x refers to the new version of the package.